lib/AppInfo/Application.php

```diff
@@ -29,30 +29,18 @@
 namespace OCA\Adminly_Dashboard\AppInfo;
 
 use OCP\AppFramework\App;
-use OCP\AppFramework\Bootstrap\IBootContext;
-use OCP\AppFramework\Bootstrap\IBootstrap;
-use OCP\AppFramework\Bootstrap\IRegistrationContext;
 
-class Application extends App implements IBootstrap
+class Application extends App
 {
-
     /** @var string */
-    public const APP_NAME = 'adminly_dashboard';
+    public const APP_ID = 'adminly_dashboard';
 
     /** @var string */
     protected $appName;
 
     public function __construct()
     {
-        parent::__construct(self::APP_NAME);
-        $this->appName  = self::APP_NAME;
```

Comment:
I would also like to keep this part with assigning `$this->appName  = self::APP_NAME;`. We aren't using it now but most likely will, and it keeps a consistent way to get the app name in the code, for example in the controller.

Reply:
Should be updated in latest commit.